test/e2e/example_k8petstore.go

```diff
@@ -118,30 +118,31 @@ T:
 	for {
 		select {
 		case <-transactionsCompleteTimeout:
-			Logf("Timeout %v reached, transactions not complete.  Breaking!", tick)
+			Logf("PetStore : Completion timeout %v reached, transactions not complete.  Breaking!", tick)
 			break T
 		case <-startupTimeout:
```

Comment:
Actually, to be honest, this seems unnecessarily complicated; why do we need a separate timeout for startup and getting all the transactions?  Let's just time out after a minute?  Saving ourselves 30 seconds doesn't seem helpful.